src/app/nation/nation.component.spec.ts

```diff
@@ -1,23 +1,26 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
-import { AlertsDeleteComponent } from '../shared/alerts/alerts-delete.component';
 import { NationComponent } from './nation.component';
 import { HttpClientModule } from '@angular/common/http';
 import { ReactiveFormsModule, FormsModule } from '@angular/forms';
+import { MaterialModule } from '../shared/material.module';
 import { FormErrorMessagesComponent } from '../shared/form-error-messages.component';
+import { Jsonp, JsonpModule } from '@angular/http';
```

Comment:
we are trying to replace these deprecated elements from newer ones. So could you explore for @angular/common/http equivalent elements